environments/production.docker-compose.yaml

```diff
@@ -0,0 +1,57 @@
+version: '3.6'
```

Comment:
I don't know if it matters but `development.docker-compose.yaml` uses version `3.8`